packages/esm-patient-chart-app/src/workspace/workspace-renderer.component.tsx

```diff
@@ -31,15 +31,15 @@ export function WorkspaceRenderer({ workspace, patientUuid, active }: WorkspaceR
     };
   }, [workspace]);
 
-  const props = React.useMemo(
+  const props = useMemo(
     () =>
       workspace && {
         closeWorkspace: workspace.closeWorkspace,
         promptBeforeClosing: workspace.promptBeforeClosing,
         patientUuid,
         ...workspace.additionalProps,
       },
-    [workspace, workspace.additionalProps, workspace.closeWorkspace, workspace.promptBeforeClosing, patientUuid],
+    [workspace, patientUuid],
```

Comment:
Does this no longer depend on `workspace.additionalProps` etc.. ? If it is a dependency it should be included in the array

Reply:
Hi @ZacButko!
`workspace.additionalProps` is a static function and the lint was giving a warning and terming it as an unnecessary dependency.